docs/irf/edisp.rst

```diff
@@ -1,11 +1,36 @@
 .. _irf-edisp:
 
-Energy dispersion
+Energy Dispersion
 =================
 
-TODO: describe / illustrate the existing classes:
+as a function of of true energy and offset angle (:ref:`gadf:edisp_2d`)
+-----------------------------------------------------------------------
+The `~gammapy.irf.EnergyDispersion2D` class represents the probability density of the energy migration 
+:math:`\mu=\frac{E_{\rm reco}}{E_{\rm true}}` as a function of true energy and offset angle from the field of view center
+(:math:`E_{\rm disp}(E_{\rm reco}|p, E)` in :ref:`irf-theory`).
 
-* `~gammapy.irf.EDispKernel`
-* `~gammapy.irf.EnergyDispersion2D`
+Its format specifications are available in :ref:`gadf:edisp_2d`
+
+This is the format in which IACT DL3 energy dispersions are usually provided, as an example:
 
 .. plot:: irf/plot_edisp.py
+    :include-source:
+
+as a function of true energy (:ref:`gadf:ogip-rmf`)
+---------------------------------------------------
+`~gammapy.irf.EDispKernel` instead represents an energy dispersion as a function of true energy only 
```

Comment:
There is a bit more to say here: the `EDispKernel` is also integrated along the migration axis, to get the RMF matrix with a given "reco" energy binning....